photon-lib/src/main/java/org/photonvision/SimPhotonCamera.java

```diff
@@ -46,22 +47,50 @@ public SimPhotonCamera(String cameraName) {
     /**
     * Simulate one processed frame of vision data, putting one result to NT.
     *
-    * @param latencyMillis
+    * @param latencyMillis Latency of the provided frame
     * @param targets Each target detected
     */
     public void submitProcessedFrame(double latencyMillis, PhotonTrackedTarget... targets) {
         submitProcessedFrame(latencyMillis, Arrays.asList(targets));
     }
 
+    /**
+     * Simulate one processed frame of vision data, putting one result to NT.
+     *
+     * @param latencyMillis Latency of the provided frame
+     * @param sortMode Order in which to sort targets
+     * @param targets Each target detected
+     */
+    public void submitProcessedFrame(double latencyMillis, PhotonTargetSortMode sortMode, PhotonTrackedTarget... targets) {
+        submitProcessedFrame(latencyMillis, sortMode, Arrays.asList(targets));
+    }
+
     /**
     * Simulate one processed frame of vision data, putting one result to NT.
     *
-    * @param latencyMillis
-    * @param tgtList List of targets detected
+    * @param latencyMillis Latency of the provided frame
+    * @param targetList List of targets detected
     */
-    public void submitProcessedFrame(double latencyMillis, List<PhotonTrackedTarget> tgtList) {
+    public void submitProcessedFrame(double latencyMillis, List<PhotonTrackedTarget> targetList) {
```

Comment:
Would it be reasonable to add a sort method which is a no-op, and default to using that? Could help collapse a tiny bit of code duplication.